js1/funkce-obory/obor-platnosti.md

```diff
@@ -18,9 +18,9 @@ if (age < 18) {
 }
 ```
 
-Zatím nebudeme řešit odkud se vzala proměnná :var[age]. Především si všimneme, že celý program obsahuje dohromady pět různých bloků kódu oddělených složenými závorkami. Pokud uvnitř nějakého bloku vytvoříme proměnnou, například :var[remains], tato proměnná je „vidět“ pouze uvnitř tohoto bloku. Tento blok se stává jejím :term{cs="oborem platnosti" en="scope"}. Jakmile její blok kódu skončí, proměnná :var[remains] zanikne a již s ní není možné pracovat.
+Zatím nebudeme řešit, odkud se vzala proměnná `age`. Především si všimneme, že celý program obsahuje dohromady pět různých bloků kódu oddělených složenými závorkami. Pokud uvnitř nějakého bloku vytvoříme proměnnou, například `remains`, tato proměnná je „vidět“ pouze uvnitř tohoto bloku. Tento blok se stává jejím :term{cs="oborem platnosti" en="scope"}. Jakmile její blok kódu skončí, proměnná `remains` zanikne a již s ní není možné pracovat.
```

Comment:
Význam `:var[*]` moc nechápu a jsem hodně pro backticky, kterým rozumí každý nástroj pro práci s Markdownem, ale možná na tuto změnu bude mít nějaký názor @podlomar.

Reply:
Jsem také hodně pro backticky, nerozumím tomu, proč zrovna proměnné označovat jinak. Ale i pokud bychom je měli označovat nějak speciálně, měly by se do stránky vypisovat jako kód. `:var` se teď vypisuje kurzívou a to je podle mne velmi matoucí.

Reply:
Já jsem původně proměnné chtěl mít nějak typograficky oddělné od normálního kódu, ale netrvám na tom. Pokud vám svorně přijde lepší mít backticky kolem proměnných, tak to klidně nahraďte v celém repozitáři.